CIP-0048/README.md

```diff
@@ -0,0 +1,249 @@
+          }
+        ]
+      },
+      // payload
+      "p": {
+        "<payload_id>": ["<data>", "<data>"], // payload0
+        "<payload_id>": ["<data>"] // payload1
+      }
+    }
+  }
+}
+```
+
+## Notes
+
+- CIP25 currently defines a 'required' image tag.
+  - CIP48 alters the use of image. Requiring the 'user' to first check if the image string matches any reference names. Then to fallback on the default behavior.
+    - example, in which the image is found by referring to payloads.
+    ```json
+    {
+      "image":"myImageNFT"
+      "refs":[
+        {
+          "name": "myImageNFT"
+          "mediaType":"image/svg+xml;utf8"
+          "src": ["refToPayloadA", "refToPayloadB"]
+        }
+      ]
+    }
+    ```
+  - The `image` tag can still be used as a thumbnail if required.
+- If there are multiple `refs` defined the references that are _NOT_ defined in the `image` tag have the same behavior as the `files` tag currently defined in CIP25
+
+# Example
+
+Mint transaction 1
+
+```json
+{
+  "721": {
+    "ext": ["cip48"],
+    "0000000000000000000000000000000000000000000000000000000A": {
+      "NFT0": {
+        "project": "CIP48 Example",
+        "name": "NFT0",
+        // references
+        "refs": [
+          {
+            "name": "NFT0",
+            "mediaType": "text/plain",
+            "src": ["0", "2"]
+          }
+        ]
+      },
+      // payload
+      "p": {
+        "0": ["Hello"],
+        "1": ["Goodbye"]
+      }
+    }
+  }
+}
+```
+
+Mint transaction 2
+
+```json
+{
+  "721": {
+    "ext": ["cip48"],
+    "0000000000000000000000000000000000000000000000000000000A": {
+      "NFT1": {
+        "project": "CIP48 Example",
+        "name": "NFT1",
+        // references
+        "refs": [
+          {
+            "name": "NFT1",
+            "mediaType": "text/plain",
+            "src": ["0", "3"]
+          }
+        ]
+      },
+      // payload
+      "p": {
+        "2": ["World"],
+        "3": ["Moon"]
+      }
+    }
+  }
+}
+```
+
+### Pseudo code walk through using the above example (using default behavior)
+
+1. Find all transactions for the given `<policy_id>`
+
+   - `found 2 mint transcations`
+
+2. Check the **ext** tag for **cip48**
+
+   - `found cip48` we now know the metadata contains references and payloads
+
+3. Iterate over each transaction. If a payload is found append that to an map or some data structure
+
+   - found 4 payloads
+   - ```js
+     payloads = { 0: "Hello", 1: "Goodbye", 2: "World", 3: "Moon" };
+     ```
+
+4. Find all `refs` (references) for the `<asset_name>` we use NFT0. Then build the data in the order defined by the `src` array.
+
+   - found 2 references
+   - ```js
+     nft0_refs = ["0", "2"];
+     nft0_data = "HelloWorld";
+     ```
+
+5. use the mediaType (mimetype) to determine the data.
+   - mediaType = "test/plain"
+   - NFT0 contains text "HelloWorld"
+
+# Backwards compatibility
+
+Handled via the use of the `"ext"` tag defined in CIP-48.
+
```

Comment:
```suggestion
```
I think we've established that the tag will not be used, right?  However compatibility would be handled is a good item to included in the Rationale section which still needs to be defined below.